appium/tests/specs/mock/composeEmail/CheckSignOnlyKey.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { MockApi } from 'api-mocks/mock';
+import { MockApiConfig } from 'api-mocks/mock-config';
+import { MockUserList } from 'api-mocks/mock-data';
+import {
+  MailFolderScreen,
+  MenuBarScreen,
+  NewMessageScreen,
+  SetupKeyScreen,
+  SplashScreen,
+} from '../../../screenobjects/all-screens';
+
+describe('COMPOSE EMAIL: ', () => {
+  it('check encrypting message for user which contains sign only key', async () => {
+    const mockApi = new MockApi();
+
+    const recipient = MockUserList.signOnlyKey;
+    const subject = 'sign only key subject';
+    const message = 'sign only key message';
+
+    mockApi.fesConfig = MockApiConfig.defaultEnterpriseFesConfiguration;
+    mockApi.ekmConfig = MockApiConfig.defaultEnterpriseEkmConfiguration;
+    mockApi.addGoogleAccount('e2e.enterprise.test@flowcrypt.com');
+    mockApi.attesterConfig = {
+      servedPubkeys: {
+        [MockUserList.signOnlyKey.email]: MockUserList.signOnlyKey.pub!,
```

Comment:
let's test original issue - user should have sign-only key and another one usable for encryption. and then we should check that app correctly chooses usable for encryption key

Reply:
variable name might be incorrect. but this key contains sign-only key plus normal key(usable for encryption)

Reply:
Got it, but having 2 different keys (not subkeys) should be more close to real situation.
We can use existing mock user (for example `robot@flowcrypt.com`) and add sign-only key as `pubSignOnly` property.